### DISCUSSION TITLE
chia coin has not arrived in my wallet.

### DISCUSSION THREAD

**Discussion by KILIC77** [Chia-Network/chia-blockchain#4104] [Support]
Although I won chia, it was not transferred to my wallet. Can I find out why. The block shows solved but not transferred to my wallet.

**Comment by KILIC77** [comment]
https://www.hizliresim.com/r9pm9z

**Reply by ghost** [reply]
Hocam selam sanırım kazandınız bende 2 haftafır uğraşıyorum bana telegramdan yada instagramdan yazar mısınız @ehmzkan

**Reply by KILIC77** [reply]
Furkan Kumanlı arka kısımda HERBALIFE NUTRITION var ücretli yardımcı oluyor ben çok anlamıyorum bu arkadaş benim bütün sistemi kurdu @kazim0690

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
